apps/AEPSampleApp/extensions/EdgeView.tsx

```diff
@@ -26,7 +26,47 @@ const EdgeView = ({navigation}: NavigationProps) => {
   function sendEvent(datasetId?: string) {
     var xdmData = {eventType: 'SampleXDMEvent'};
     var data = {free: 'form', data: 'example'};
-    var experienceEvent = new ExperienceEvent(xdmData, data, datasetId);
+
```

Comment:
Have the methods here for testing purpose.  Will remove or add more buttons for them in the Edge view later.